orc8r/cloud/go/services/bootstrapper/servicers/registration/storage.go

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package registration
+
+import (
+	"github.com/pkg/errors"
+
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/cloud/go/services/bootstrapper"
+	"magma/orc8r/cloud/go/storage"
+	merrors "magma/orc8r/lib/go/errors"
+	"magma/orc8r/lib/go/protos"
+)
+
+const networkWildcard = "*"
```

Comment:
https://github.com/magma/magma/blob/2104a1c17c4a773dca84d5397665e2db89d565a2/orc8r/cloud/go/services/accessd/storage/storage_blobstore.go#L34-L36

And as a bonus, could move that placeholder to a const in a shared orc8r pkg

Reply:
ok created tix for me later: https://github.com/magma/magma/issues/10533

Reply:
Let's go ahead and make it "placeholder_network" in this PR, since changing things like this later generally requires a data migration

Reply:
Hm I copied this one: https://github.com/magma/magma/blob/master/orc8r/cloud/go/services/tenants/servicers/storage/storage.go#L15
`const networkWildcard = "*"`

Would you like me to call it "placeholder_network" in this PR?

Reply:
placeholder_network